### ISSUE TITLE
Conversion error while converting yolov8n.ONNX model

### ISSUE THREAD

**Issue by Krishna18698** [luxonis/blobconverter#55]
OS: Windows 10
Openvino version:2022.3.0

This is the error i get.
Unknown model format! Cannot find reader for model format: xml and read the model: /tmp/blobconverter/148c418a26bc4ac598f8702157228fe1/yolov8n/FP16/yolov8n.xml. Please check that reader library exists in your PATH.

anyway to solve this

thanks,

**Comment by Krishna18698** [comment]
after getting the best.pt file after the training
I converted it into ir format and then tried to convert it into blob it finally worked giving errors 5 or six times
there is that error i get 
[1844301051D1420E00] [1.4] [20.593] [XLinkOut(4)] [error] Message has too much metadata (138317B) to serialize. Maximum is 51200B. Dropping message
when i run the model tho
i followed the notebook only thing is i used the coco dataset besides that everything is the same
when i try to convert the pt file directly to the blob using luxonis tool 
i get that error saying contact developers
I have posted about the same in the
https://github.com/luxonis/depthai-ml-training/blob/master/colab-notebooks/YoloV8_training.ipynb 
issues
with the pt file and place i got the dataset from.
